src/style.css

```diff
@@ -0,0 +1,110 @@
+html {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
move it to the * selector

Reply:
it meant move `box-sizing` to `*`

leave font styles in `html`